components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -97,21 +97,17 @@ func TestGetPolicy(t *testing.T) {
 		},
 		"policy with two statements found": func(t *testing.T) {
 			ctx := context.Background()
-			// Note (sr): Is it ugly to put the SQL statement in here, with no
```

Comment:
[note] All of these code changes are refactors to use the helper I added except where noted.